### PR TITLE
toggle form created

### PR DESCRIPTION
**Pull request by ShreyaDhiman24**
## Related Issue
#494 

Closes: #494 

## Description of Changes

[Screencast from 18-07-23 01:18:03 AM IST.webm](https://github.com/anmode/grabtern-frontend/assets/98320971/74821d6e-6baa-4413-b77f-34ae868d5ecc)

## Checklist:

- [ ] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.
- [x] All of them

### CODE REVIEW

File: pages/mentorLogin.page.js [hunk 1]
```diff
@@ -117,101 +118,199 @@ function mentorLogin() {
...
-                aria-required="true"
-              />
-              <div
-                className="tw-absolute tw-inset-y-0 tw-right-0 tw-flex tw-px-4 tw-text-gray-600 tw-top-16"
-                onClick={togglePasswordVisibility}
-                aria-label={
-                  isPasswordVisible ? "Hide Password" : "Show Password"
-                }
+
+      <div className="login-form d-flex flex-column bg-white tw-mb-10">
```
Comment (anmode) [inline pages/mentorLogin.page.js:139]: To remove white space, modify padding in the `login-form` class

**Review by anmode — CHANGES_REQUESTED** [review]
The functionality left, on submit API are different for the mentor as well as for the user too.
this case applies to google login too.

Mobile optimization is also left

**Comment by anmode** [comment]
@ShreyaDhiman24 ?

**Comment by ShreyaDhiman24** [comment]
@anmode I have fixed all the issues that were there in the css..you can checkout..!

**Comment by anmode** [comment]
cool@

**Pull request merged**